src/main/java/org/jenkinsci/plugins/github/internal/GitHubClientCacheOps.java

```diff
@@ -96,7 +95,6 @@ public static Path getBaseCacheDir() {
      *
      * @param configs active server configs to exclude caches from cleanup
      */
-    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE")
```

Comment:
This was needed in a previous version of SpotBugs, but it is not needed in the very latest version of SpotBugs (pulled in via the latest plugin parent POM). The latest version actually complains about an unnecessary suppression, so remove it here to fix that warning.